src/utils/navigation.ts

```diff
@@ -23,4 +24,9 @@ function getHash(url: string) {
 function removeChars(subStr: string) {
   return (str: string) => str.replace(subStr, "");
 }
-export const getToken = flow(getHash, removeChars("#token="));
+export const getToken = flow(
+  getHash,
+  removeChars("#token="),
+  O.fromNullable,
+  O.getOrElse(() => sessionStorage.getItem("bearerAuth"))
```

Comment:
```suggestion
  O.of,
  O.filter(h => h !== "")
  O.getOrElse(() => sessionStorage.getItem("bearerAuth"))
```

* `URL.hash` cannot be null, so we can just `O.of`
* `filter` is necessary if we want to check whether there is no fragment